src/Sylius/Bundle/CoreBundle/Tests/Twig/ChannelUrlExtensionTest.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\Tests\Twig;
+
+use PHPUnit\Framework\TestCase;
+use Sylius\Bundle\CoreBundle\Twig\ChannelUrlExtension;
+use Sylius\Component\Channel\Context\ChannelContextInterface;
+use Sylius\Component\Channel\Model\Channel;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\RequestStack;
+use Symfony\Component\HttpFoundation\UrlHelper;
+
+class ChannelUrlExtensionTest extends TestCase
+{
+    public function testGenerateChannelUrlWithChannelHostname(): void
```

Comment:
From what I remember, we are using phpspec notation for PHPUnit tests as well
```suggestion
    /** 
     * @test 
     */
    public function it_generates_channel_url_with_channel_hostname(): void
```

Reply:
True. I think I copied it from somewhere else...